pkgs/development/libraries/mesa/stubs.nix

```diff
@@ -61,9 +61,9 @@ stdenv.mkDerivation {
       cat <<EOF >$dev/lib/pkgconfig/$name.pc
     Name: $name
     Description: $lib library
-    Version: ${mesa.version}
+    Version: ${libglvnd.version}
     Libs: -L${libglvnd.out}/lib -l$lib
-    Cflags: -I${mesa.dev}/include -I${libglvnd.dev}/include
+    Cflags: -I${libglvnd.dev}/include
```

Comment:
This is the other part that's difficult to review. I haven't diffed `mesa.dev` and `libglvnd.dev` yet but if `mesa.dev` provides additional (or even just slightly different headers, e.g. due to older versions) this could potentially cause a lot of breakages (and due to the version change alone). Is there any official documentation for this and do we know what other distros do?